package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "harp": "harp ",
     "harp:compile": "npm run harp compile styleguide dist",
-    "start": "harp server styleguide",
+    "start": "concurrently 'harp server styleguide' 'browser-sync start --config bs-config.js'",
+    "bs-start": "browser-sync start --config bs-config.js",
```

Comment:
was only keeping it in case someone found it useful :) ,but yes it is not what I would consider needed. I'll remove it. we can add something like that later if it becomes necessary.